src/app-host/app-host.js

```diff
@@ -57,38 +100,6 @@ function getCordova() {
     return cordova;
 }
 
```

Comment:
Plugins simulation (app-host side) is loaded and initialized without being attached to the overall initialization. That could cause issues like requesting exec calls to sim-host side when sim-host is `ready` but plugins are not ready yet because they are initializing. So this exec calls in sim-host fails, because there is no handler, when actually it is.